modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm

```diff
@@ -485,6 +487,17 @@
 							create_cum_decal = TRUE
 							visible_message(span_userlove("[src] shoots their sticky load onto the [target_human]!"), \
 								span_userlove("You shoot string after string of hot cum onto [target_human]!"))
+						else if(climax_into_choice == "face")
+							visible_message(span_userlove("[src] shoots their sticky load onto the [target_human] face!"), \
+								span_userlove("You shoot string after string of hot cum onto [target_human] face!"))
+							if(target_human.GetComponent(/datum/component/cumfaced))
+								if(!target_human.GetComponent(/datum/component/cumfaced/big))
+									target_human.TakeComponent(/datum/component/cumfaced)
+									target_human.AddComponent(/datum/component/cumfaced/big)
```

Comment:
I'll have to look into this more to give a better suggestion but I'm instinctively not a fan of the double `GetComponent`

Reply:
i think i found a solution